### ISSUE TITLE
linint2 output is a dask array

### ISSUE THREAD

**Issue by brianpm** [NCAR/geocat-f2py#86]
Using `linint2` I find that if my inputs are Xarray DataArrays, I am getting output that is correct except that it is a dask array. This behavior does not seem intuitive to me, and I don't see it in the [documentation](https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.linint2.html#geocat.comp.linint2). To me, if I pass arguments that are in memory, I feel like I'm telling Geocat that I want the computation to be done immediately. This behavior breaks downstream code in some cases. For example, in my script I was using `.item()` to log scalar values, but that raises `NotImplementedError: 'item' is not yet a valid method on dask arrays`. 

In `linint2`, I see at the end that this behavior is built in, but the comments say that it will return a DataArray. That's slightly misleading when it is a DataArray that is really a dask array: 

```python
    # If input was xarray.DataArray, convert output to xarray.DataArray as well
    if is_input_xr:
        fo = xr.DataArray(fo, attrs=fi.attrs, dims=fi.dims, coords=fo_coords)
    # Else if input was numpy.ndarray, convert Dask output to numpy.ndarray with `.compute()
    else:
        fo = fo.compute()

    return fo
```

It would be nice to have a kwarg to specify to call `compute()`.... e.g., 

```python
    if (is_input_xr and output_dask):
        fo = xr.DataArray(fo, attrs=fi.attrs, dims=fi.dims, coords=fo_coords)
    elif (is_input_xr and output_xr)
        fo = (xr.DataArray(fo, attrs=fi.attrs, dims=fi.dims, coords=fo_coords)).compute()
    # Else if input was numpy.ndarray, convert Dask output to numpy.ndarray with `.compute()
    else:
        fo = fo.compute()
```

**Comment by erogluorhan** [comment]
Hi @brianpm , thanks for great feedback! We made a recent change in our input-output structure, so this case is due that change. We can fix this just as you recommended, then there won't be any issues with the regular (unchunked) xarray.DataArray inputs.

**Comment by brianpm** [comment]
That sounds good, @erogluorhan , thanks for the reply!

**Comment by erogluorhan** [comment]
Hi @brianpm , this is addressed in PR #87  and that work is already merged. You may either update your GeoCAt-f2py installation through Github repo or wait for our April release to get through conda. Once you check the bugfix and confirm it is fixed, please feel free to close this issue.

**Comment by brianpm** [comment]
Awesome! I'll wait for the conda version (otherwise I'm sure I'll get myself into trouble). I looked at the PR, though, and it seems great, so I will close the issue. Thank you!!
